### PR TITLE
Refactor/presentation files

### LINKED ISSUE #475: Update presentation files with typography and spacing
Final list of presentation pages (and their statuses):

| Fully Removed  | Partfully Removed | Not Changed |
| ------------- | ------------- | ------------- |
| Dashboard, DashboardBottomWidgets, SetupProfileTab, UsersMembersTab | DashboardInfoWidgets, DataVisualisation, FeaturesTab | - |
| - | - | Charts, ProgressBars, CircleBars, ColorfulBars, HorizontalBars |
| FormWizard | - | FormElements, MediumEditor |
| BadgeColumn | - | Table |
| Buttons, Collapse, Color Pickers, Filters, Modals | Cards, Grid, ToastPositionPicker, SetsList, IconSet, Spinners | Icons, Notifications, Tags, File Upload, Tree view, Spacing |
| - | Extra | - |
| - | AuthLayout, Login, Signup | - |
| BubbleMap, GoogleMap, LineMap, LeafletMap | BubbleMapsPage, GoogleMapsPage, LeafletMapsPage, LineMapsPage, YandexMapsPage | - |
| - | 404PagesPage, MadeByComponent, VuesticPageNotFoundCustom, VuesticPageNotFoundLargeText, VuesticPageNotFoundSearch, VuesticPageNotFoundSimple, VuesticPageNotFoundSadWallpaper | - |

### PR DESCRIPTION
**Pull request by AlexanderBoriskin**
close #475

### CODE REVIEW

File: src/components/pages/404-pages/VuesticPageNotFoundLargeText.vue [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
-          <a href="mailto:hello@epicmax.co">hello@epicmax.co</a>
+          </span>
+          <a class="mt-4" href="mailto:hello@epicmax.co">hello@epicmax.co</a>
         </div>
       </div>
-      <div
-        class="vuestic-page-not-found-large-text__wallpaper-button">
+      <div class="vuestic-page-not-found-large-text__wallpaper d-flex
+        justify--center align--start pt-4 pb-3 overflow--hidden">
         <va-button :to="{ name: 'dashboard' }">
```
Comment (asvae) [inline src/components/pages/404-pages/VuesticPageNotFoundLargeText.vue]: Button not clickable.

image

File: src/components/ui/collapse/Collapse.vue [hunk 2]
```diff
@@ -8,8 +8,8 @@
             <div slot="body">
               <div class="va-row">
                 <div class="flex md6">
-                  <div class="collapse-page__content">
-                    <h4 class="collapse-page__content__title">February 2018</h4>
+                  <div class="collapse-page__content pa-4">
```
Comment (asvae) [inline src/components/ui/collapse/Collapse.vue]: You can probably remove `collapse-page__*` styles. They're informative, but serve no styling purpose.

File: src/components/ui/filters/Filters.vue [hunk 3]
```diff
@@ -1,55 +1,54 @@
...
+          <filter-bar
+            v-model="name"
+            class="filters-page__filter-bar"
+            label="Name"
+          />
+        </div>
+        <div class="flex md3 offset--md1">
+          <filter-bar
+            v-model="email"
+            class="filters-page__filter-bar"
```
Comment (asvae) [inline src/components/ui/filters/Filters.vue]: Useless classes

File: src/components/ui/grid/Grid.vue [hunk 4]
```diff
@@ -5,17 +5,17 @@
         <div>
           <div class="va-row">
             <div class="flex xs12">
-              <div class="_green-rectangle">
+              <div class="ui-grid__container d-flex justify--center align--center">
```
Comment (asvae) [inline src/components/ui/grid/Grid.vue]: I sugges renaming file to `UiGrid.vue` to correspond.

File: src/components/ui/grid/Grid.vue [hunk 5]
```diff
@@ -49,26 +49,26 @@
       <vuestic-widget headerText="Mobile and Desktop Grid">
         <div class="va-row">
           <div class="flex xs12 md4">
-            <div class="_green-rectangle">xs12 md4</div>
+            <div class="ui-grid__container d-flex justify--center align--center">xs12 md4</div>
```
Comment (asvae) [inline src/components/ui/grid/Grid.vue]: You can move aligns into `ui-grid__container`.
That's what mixins are for.

File: src/vuestic-theme/vuestic-components/vuestic-profile-card/VuesticProfileCard.vue [hunk 6]
```diff
@@ -1,11 +1,13 @@
 <template>
   <div class="vuestic-profile-card flex align--center">
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-profile-card/VuesticProfileCard.vue]: d-flex?

File: src/vuestic-theme/vuestic-components/vuestic-profile-card/VuesticProfileCard.vue [hunk 7]
```diff
@@ -1,11 +1,13 @@
...
-    <div class="photo">
+    <div class="vuestic-profile-card__photo">
       <img :src="photoSource"/>
     </div>
-    <h5 class="name">{{name}}</h5>
-    <span class="location"><span
-      class="icon glyphicon glyphicon-map-marker"></span> {{location}}</span>
+    <div class="vuestic-profile-card__info">
+      <h5 class="name">{{name}}</h5>
+      <span class="location"><span
```
Comment (asvae) [inline src/vuestic-theme/vuestic-components/vuestic-profile-card/VuesticProfileCard.vue]: You can BEMify this similarly to what you did with `photo` and `info`.

File: src/vuestic-theme/vuestic-sass/resources/_variables.scss [hunk 8]
```diff
@@ -35,6 +35,9 @@ $dark-gray: #282828;
 $gray: #adb3b9;
 $dark-blue: #0045b6;
 $text-gray: #b4b4b4;
+$text-highlighted: #fff3d1;
```
Comment (asvae) [inline src/vuestic-theme/vuestic-sass/resources/_variables.scss]: These variables are declared 2 times.

File: src/components/admin/app-navbar/components/dropdowns/LanguageDropdown.vue [hunk 9]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="language-dropdown flex-center">
+  <div class="language-dropdown flex-center spacer">
```
Comment (asvae) [inline src/components/admin/app-navbar/components/dropdowns/LanguageDropdown.vue]: `spacer` is not supposed to be used that way.

File: src/components/dashboard/features-tab/FeaturesTab.vue [hunk 10]
```diff
@@ -1,44 +1,44 @@
 <template>
   <div class="overview-tab dashboard-tab">
     <div
-      class="d-flex overview-row justify--space-around justify--start">
+      class="va-row d-flex overview-row justify--space-around justify--start">
 
-      <div class="overview-col">
-        <div class="overview-item">
-          <div class="overview-icon-container">
+      <div class="xs12 overview-col">
```
[the remark this answers is not in the document]
Comment (asvae) [reply src/components/dashboard/features-tab/FeaturesTab.vue]: Still does nothing.

File: src/components/pages/404-pages/VuesticPageNotFoundLargeText.vue (hunk 1 above)
Comment (AlexanderBoriskin) [reply src/components/pages/404-pages/VuesticPageNotFoundLargeText.vue]: Not reproduced

File: src/components/auth/login/Login.vue [hunk 11]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="login">
-    <h2>{{ $t('auth.welcome') }}</h2>
+    <div class="display-2 text--center my-4">{{ $t('auth.welcome') }}</div>
```
Comment (asvae) [inline src/components/auth/login/Login.vue]: No problem keeping h2 here for bots etc.

**Pull request merged**